modules/calc/src/test/java/com/opengamma/strata/calc/runner/CompositeCalculationFunctionsTest.java

```diff
@@ -33,12 +33,11 @@ public void compose() {
     CalculationFunctions fns2 = CalculationFunctions.of(fn1b, fn2);
     CalculationFunctions composed = fns1.composedWith(fns2);
 
-    assertThat(composed.getFunction(new Target1())).isEqualTo(fn1a);
-    assertThat(composed.getFunction(new Target2())).isEqualTo(fn2);
+    assertEquals(composed.getFunction(new Target1()), fn1a);
```

Comment:
Why change assertions? I prefer AssertJ but don't think it's worth switching every test I touch.